src/rust/inetstack/protocols/layer2/ethernet2/frame.rs

```diff
@@ -34,30 +50,27 @@ impl Ethernet2Header {
         }
     }
 
-    pub fn compute_size(&self) -> usize {
```

Comment:
We no longer need this function because we only use the ethernet header size internally now